qa/rpc-tests/p2p-acceptblock.py

```diff
@@ -278,8 +278,15 @@ def run_test(self):
 
         # 7. Send the missing block for the third time (now it is requested)
         test_node.send_message(msg_block(blocks_h2f[0]))
-
         test_node.sync_with_ping()
+
+        # Wait for the reorg to complete. It can be slower on some systems.
+        while self.nodes[0].getblockcount() != 290:
+            time.sleep(1)
+            j = j + 1
+            if (j > 60):
```

Comment:
sleeping is not an accurate time measure. A better way to do this is to get a timestamp from the system and then compare with the current timestamp. If the difference is greater than 60, then bail out.

Otherwise, the change to this test can go in in their own PR today.